.github/workflows/scheduled-benchmarks-self-hosted.yaml

```diff
@@ -90,12 +90,12 @@ jobs:
           sudo chmod 777 ~/.kube
 
       - name: Setup Kubernetes
-        uses: manusa/actions-setup-minikube@v2.7.1
+        uses: manusa/actions-setup-minikube@v2.6.1
         with:
           minikube version: 'v1.23.2'
           kubernetes version: 'v1.23.2'
           driver: docker
-          start args: --disk-size 10000mb
+          start-args: --disk-size 10000mb
```

Comment:
Why the difference in the action each time while setting up kubernetes?

Reply:
This one didn't work too, I'll change it to curl based method